cmd/markdown_help.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import _ "embed"
+
+//go:embed markdown/atmos_terraform_usage.md
+var terraform string
```

Comment:
Let's keep that variables similar to the filenames. That way we support more than just usage, since this package is not only for usage